lib/quoters/WebhookQuoter.ts

```diff
@@ -177,12 +194,13 @@ export class WebhookQuoter implements Quoter {
       }
 
       // RFQ provider response failed validation
-      if (validationError) {
+      if (validationError || validatePermissionedTokensError) {
+        const error = validationError || validatePermissionedTokensError;
         metric.putMetric(Metric.RFQ_FAIL_VALIDATION, 1, MetricLoggerUnit.Count);
         metric.putMetric(metricContext(Metric.RFQ_FAIL_VALIDATION, name), 1, MetricLoggerUnit.Count);
```

Comment:
nit: should we track permissioned token related errors separately? I don't know if we'd want to alarm on them, for example

Reply:
I don't see any alarms on this event. I think we're good to log them together.